app/general/models.py

```diff
@@ -1,63 +1,80 @@
 from django.core.validators import FileExtensionValidator
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 from simple_history.models import HistoricalRecords
 
 
 class Project(models.Model):
-    name = models.CharField(max_length=200)
-    url = models.URLField(max_length=200, blank=True, verbose_name="URL")
-    logo = models.ImageField(upload_to="projects/logos/", blank=True)
-    start_date = models.DateField(blank=True, null=True)
-    end_date = models.DateField(blank=True, null=True)
+    name = models.CharField(max_length=200, verbose_name=_("name"))
+    url = models.URLField(max_length=200, blank=True, verbose_name=_("URL"))
+    logo = models.ImageField(upload_to="projects/logos/", blank=True, verbose_name=_("logo"))
+    start_date = models.DateField(blank=True, null=True, verbose_name=_("start date"))
+    end_date = models.DateField(blank=True, null=True, verbose_name=_("end date"))
     institution = models.ForeignKey(
-        "Institution", on_delete=models.CASCADE, blank=True, verbose_name="institution"
+        "Institution", on_delete=models.CASCADE, blank=True, verbose_name=_("institution")
     )
-    subjects = models.ManyToManyField("Subject", blank=True)
-    languages = models.ManyToManyField("Language", blank=True)
+    subjects = models.ManyToManyField("Subject", blank=True, verbose_name=_("subjects"))
+    languages = models.ManyToManyField("Language", blank=True, verbose_name=_("languages"))
 
     # added simple historical records to the model
     history = HistoricalRecords()
 
+    class Meta:
+        verbose_name = _("Project")
+        verbose_name_plural = _("Projects")
+
     def __str__(self):
         return self.name
 
 
 class Institution(models.Model):
-    name = models.CharField(max_length=200, unique=True)
-    abbreviation = models.CharField(max_length=200)
-    url = models.URLField(max_length=200, blank=True, verbose_name="URL")
-    email = models.EmailField(max_length=200, blank=True)
-    logo = models.ImageField(upload_to="institutions/logos/", blank=True)
+    name = models.CharField(max_length=200, unique=True, verbose_name=_("name"))
+    abbreviation = models.CharField(max_length=200, verbose_name=_("abbreviation"))
+    url = models.URLField(max_length=200, blank=True, verbose_name=_("URL"))
+    email = models.EmailField(max_length=200, blank=True, verbose_name=_("email"))
+    logo = models.ImageField(upload_to="institutions/logos/", blank=True, verbose_name=_("logo"))
 
     # added simple historical records to the model
     history = HistoricalRecords()
 
+    class Meta:
+        verbose_name = _("institution")
```

Comment:
Please start this one with a capital letter as well.

Reply:
done